teams/nixos_release.tt

```diff
@@ -0,0 +1,41 @@
+[% WRAPPER layout.tt hideTitle=1 title="NixOS Release Team" menu='nixos' %]
+
+<h1>NixOS Release Team</h1>
+
+<div class="lead">
+  For each release there are two release managers. After each release the
+  release manager having managed two releases steps down and the release
+  management team of the last release appoints a new release manager.
+</div>
+
+<p>
+  This makes sure a release management team always consists of one release
+  manager who already has managed one release and one release manager being
+  introduced to their role, making it easier to pass on knowledge and
+  experience.
+</p>
+
+<h2>Members</h2>
+
+<ul>
+  <li><a href="https://github.com/worldofpeace">@worldofpeace</a></li>
+  <li><a href="https://github.com/disassembler">@disassembler</a></li>
+</ul>
+
+<h2>Responsibilities</h2>
+
+<ul>
+  <li>manage the release process</li>
+  <li>start discussions about features and changes for a given release</li>
+  <li>create a roadmap</li>
+  <li>release in cooperation with Eelco Dolstra</li>
+  <li>decide which bug fixes, features, etc... get back ported after a release</li>
+</ul>
+
+<h2>Release process</h2>
+
+More about it you can
+<a href="https://nixos.org/nixos/manual/index.html#release-managers">
+  read in the manual</a>
```

Comment:
This reads funny to me. How about, more information can be found in the nixos manual?